ontology/yaml/resources/METERS/entity_types/ABSTRACT.yaml

```diff
@@ -67,3 +67,13 @@ PVCM:
   - phase1_neutral_line_voltage_sensor
   - phase2_neutral_line_voltage_sensor
   - phase3_neutral_line_voltage_sensor
+  
+PWMRDM2X:
+  description: "Energy monitoring from Electrical Grid and Diesel Generator with run hours monitoring."
```

Comment:
If these are coming from two separate, distinct sources then they should have distinct names. Which field serves which source? Please adjust field names. something like `generator_energy_accumulator` and `electrical_grid_energy_accumulator`.